.github/workflows/build.yml

```diff
@@ -16,17 +16,28 @@ jobs:
     - name: Setup dependencies
       run: |
         sudo apt update
-        sudo apt install texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended xsltproc
+        sudo apt install texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended xsltproc ghostscript cm-super librsvg2-bin inkscape
       
     - name: Build the document
-      run: make biblio forcetex
+      run: make role_diagram.pdf biblio forcetex
       
     - name: Check the output
       run: |
         test -f DataLink.pdf
         test -f DataLink.bbl
         
+    - name: Build the implementation note
+      run: make --makefile=MakefileImp forcetex
```

Comment:
I think this needs the biblio step as the generated pdf has (?) for all in text citations and has no references section

Reply:
added